src/services/LoggingService.js

```diff
@@ -51,14 +51,15 @@ class LoggingService {
       updatedCustomAttributes = Object.assign({}, responseAttributes, customAttributes);
       processedError = new Error(`API request failed: ${error.response.status} ${responseAttributes.errorUrl} ${data}`);
     } else if (error.request) {
+      const errorMessage = error.request.responseText || error.message;
       const requestAttributes = {
         errorType: 'api-request-error',
         errorStatus: error.request.status,
         errorUrl: error.request.responseURL,
-        errorData: error.request.responseText,
+        errorData: errorMessage,
```

Comment:
@douglashall: Would it help you to add something like: `errorRequest: fixErrorLength(JSON.stringify(error.request))` to try to see what is there?